lib/jekyll-github-metadata/metadata_drop.rb

```diff
@@ -48,6 +48,7 @@ def keys
       def_delegator :repository, :tagline,                     :project_tagline
       def_delegator :repository, :owner_metadata,              :owner
       def_delegator :repository, :owner,                       :owner_name
+      def_delegator :repository, :owner_displayname,           :owner_displayname
```

Comment:
How about `owner_display_name` throughout?

Username and metadata do make sense as single words.

But it would be "display name" as two words not "displayname", outside of the code. Therefore underscore in the code would split the words well.